miniapps/adjoint/cvsRoberts_ASAi_dns.cpp

```diff
@@ -221,11 +221,10 @@ int main(int argc, char *argv[])
    // Perform time-integration (looping over the time iterations, ti, with a
    // time-step dt).
    bool done = false;
-   for (int ti = 0; !done; )
+   for ( ; !done; )
```

Comment:
```suggestion
   while(!done)
```
I find this marginally quicker to digest what is going on, but feel free to ignore this suggestion.

Reply:
Agree